src/Microsoft.AspNetCore.Mvc.Testing/CultureReplacerMiddleware.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Testing;
+
+namespace Microsoft.AspNetCore.Mvc.Testing
+{
+    /// <summary>
+    /// A middleware that ensures web sites run in a consistent culture. Currently useful for tests that format dates,
+    /// times, or numbers. Will be more useful when we have localized resources.
+    /// </summary>
+    public class CultureReplacerMiddleware
```

Comment:
It would be nice to pull a bunch of this stuff out of the `Mvc.Testing` namespace into something more obscure. You really only need a few of these classes.